### ISSUE TITLE
docs: installation instructions on the website

### ISSUE THREAD

**Issue by andela-anandaa** [facebook/rocksdb#3565]
We need to add installation instructions as part of the _Getting Started_ section of the website. What do you think? I could go ahead and do a PR on this.

**Comment by gfosco** [comment]
Agreed and was planning to work on that soon-ish.  Happy to review a PR if you'd like to get involved!

**Comment by ramvadiv** [comment]
Added a note/link to INSTALL.md in the Getting started section of the wiki at https://github.com/facebook/rocksdb/wiki
